### DISCUSSION TITLE
Bot Stop to play and leave

### DISCUSSION THREAD

**Discussion by Sirius3331** [eritislami/evobot#960] [Q&A] [answered]
Hello guys,
I have a little problem with my bot which uses evobot's codes. The bot is playing normally my musics in the playlist and stops playing and leaved the Voice Channel. Somebody know how to fix it?
Kind regards :)

**Comment by Sirius3331** [comment, marked as answer]
I found it uot for myself. Thanks anyway that I can ask you here my problems. I wish you a great day.

**Reply by YarpenDev** [reply]
What did you do? Mine sometimes leaves without any command or just stops playing.
